src/Routes/RosPage/RosPage.js

```diff
@@ -181,9 +198,11 @@ class RosPage extends React.Component {
                 stateFilterValue: activeStateFilters
             });
         }
+        // Fixme : how to delete os filter here?
     }
 
     getActiveFilterConfig = () => {
+        // Fixme : how to define os filter here?
         const activeFilters = this.state.stateFilterValue.map((value)=> ({ name: value }));
```

Comment:
@apuntamb as we discussed can you update this code to create an array of object for OSFilter value. This method is responsible for creating the active filters chips on the UI. Also feel free to refactor the variable from `activeFilters` to `stateActiveFilters` 

the final array which we want to return will be:
`            [{
                category: 'State',
                chips: stateActiveFilters
            },
            {
                category: 'Operating Systems',
                chips: osActiveFilters
            }]
  `

You might need to add few conditions to check if there are any OS or State filter values selected and create the above array based on it. 
Happy to do call to discuss further on this.
Also for reference go through link here: https://github.com/RedHatInsights/frontend-components/blob/4c5456bd2b27497fbc8849bf5aad99be1d6fb755/packages/components/doc/primary_toolbar.md